src/main/java/cc/hyperium/purchases/PurchaseApi.java

```diff
@@ -60,7 +60,11 @@ private PurchaseApi() {
         for (EnumPurchaseType enumPurchaseType : EnumPurchaseType.values()) {
             purchaseClasses.putIfAbsent(enumPurchaseType, DefaultCosmetic.class);
         }
-        getPackageAsync(UUIDUtil.getClientUUID(), hyperiumPurchase -> System.out.println("Loaded self packages: " + hyperiumPurchase.getResponse()));
+        try {
+            getPackageAsync(UUIDUtil.getClientUUID(), hyperiumPurchase -> System.out.println("[Packages] Loaded self packages: " + hyperiumPurchase.getResponse()));
+        } catch (NullPointerException ex) {
+            getPackageAsync(UUIDUtil.getClientUUID(), hyperiumPurchase -> System.out.println("[Packages] Loaded self packages: " + hyperiumPurchase.getResponse()));
+        }
```

Comment:
What's the point of this? It just repeats the method call even if there's an error?

Reply:
It happens for most people, just wanted to prevent a stack trace from being printed